src/IssueCreator/IssueManager.cs

```diff
@@ -104,17 +105,33 @@ public async Task<bool> AddIssueToEpicAsync(long repoId, int epicNumber, long re
             }
         }
 
-        public async Task SetIssueEstimateAsync(long repoId, int issueNumber, int estimate)
+        public async Task<bool> SetIssueEstimateAsync(long repoId, int issueNumber, int estimate)
         {
-            if (estimate > 0)
+            try
+            {
+                if (estimate > 0)
+                {
+                    await _zenHubClient.GetIssueClient(repoId, issueNumber).SetEstimateAsync(estimate);
+                }
+                return true;
+            }
+            catch
```

Comment:
Maybe at least log issues somewhere, like a file or even hidden pain of list of strings?

Reply:
That's a great point! I'll do that in this PR.